src/pages/works/[slug].tsx

```diff
@@ -41,7 +42,7 @@ export default function Work({
 }
 
 export async function getStaticProps({ params }: { params: any }) {
-  const { url, basePath } = process.env
+  const { url, basePath, cloverOptions } = process.env
```

Comment:
We don't need to send this through getStaticProps on the static side. We can probably just peel it off of `config` like using our useCanopyState() hook.

```jsx
const { canopyState: { config } } = useCanopyState();
const { cloverViewer } = config?.viewer;
```

See how we did in `src/pages/search.tsx`

https://github.com/canopy-iiif/canopy-iiif/blob/2d933b086be297a4c38496c72c0247d4796562b4/src/pages/search.tsx#L28-L31

https://github.com/canopy-iiif/canopy-iiif/blob/2d933b086be297a4c38496c72c0247d4796562b4/src/pages/search.tsx#L54